### ISSUE TITLE
FindGlib.cmake

### ISSUE THREAD

**Issue by aspeteRakete** [ofnode/of#9]
Hey, right now I'm trying to get the GStreamer support on OSX one step further.
I get linker errors:
````
Undefined symbols for architecture x86_64:
  "_g_object_get", referenced from:
````
The Script doesn't check for GObject. Does this work on Linux?
I found a more complete FindGLib.cmake at https://github.com/WebKit/webkit/blob/master/Source/cmake/FindGLIB.cmake
Would it be ok to swap it with the webkit one?

**Comment by aspeteRakete** [comment]
Actually the other script doesn't work. I'll try to improve the existing one.

**Comment by procedural** [comment]
Undefined references are saying that "I've linked all libraries, but some libraries which define "_g_object_get" are missing". Which library define `_g_object_get`?..

**Comment by aspeteRakete** [comment]
gobject. I added this to the script. will upload later.

**Comment by procedural** [comment]
I made this alias on Linux which helps me search undefined functions, you can find it handy :)

```bash
alias defs='function _defs(){ for LIB in `find $2 \( -name "*.a" -o -name "*.dll" \) 2>&1`; do if nm -p $LIB | grep $1; then echo -e "These are found in \033[1;45m$LIB\033[0m\n"; fi; done; }; _defs'
```

```bash
$ defs _g_object_get /usr/lib

                 U g_object_get_property
                 U g_object_get_qdata
0000000000002ef0 T g_object_get_type
0000000000003e30 T g_object_get_valist
0000000000004290 T g_object_get
00000000000047a0 T g_object_get_property
00000000000054d0 T g_object_get_qdata
00000000000058f0 T g_object_get_data
These are found in /usr/lib/x86_64-linux-gnu/libgobject-2.0.a
```

**Comment by aspeteRakete** [comment]
Can you check if the FindGlib.cmake on feature-gst-osx branch works on Linux?
Cool alias you got there, have to tweak it. Right now there is something wrong under osx.

**Comment by procedural** [comment]
Yep, it compiles. Does changing [`ofGstUtils.cpp`](https://github.com/ofnode/of/commit/b0cd240) necessary for fixing compile errors? I change vanilla openFrameworks only in case of compile errors, any feature improvements I keep locally on my machine or pull request to official repository, OF is not really a fork of openFrameworks :stuck_out_tongue:

**Comment by aspeteRakete** [comment]
It seems the Videograbber won't work:
http://comments.gmane.org/gmane.comp.video.gstreamer.devel/46076

**Comment by procedural** [comment]
@aspeteRakete if you don't mind, I synced with master and moved Gstreamer feature to a new branch: https://github.com/ofnode/of/tree/feature-osx-gstreamer
